components/Form/FormURL.tsx

```diff
@@ -28,30 +28,32 @@ interface FormURLProps {
     setFormData(data: Record<string, any>): any;
 }
 const FormURL = ({ id, title, formData, setFormData }: FormURLProps) => {
-    const onSpacePressed = (e: KeyboardEvent<HTMLInputElement>) => {
+    const [error, setError] = useState(""); // Add this line to initialize the error state
+
+    const onSpacePressed = (e: React.KeyboardEvent<HTMLInputElement>) => {
         if (e.key === " ") {
             e.preventDefault();
-            setFormData((data: Record<string, any>) => ({
-                ...data,
-                urls: [...data.urls, (e.target as HTMLInputElement).value.trim()].filter((x) => x.length > 0),
-            }));
-            (e.target as HTMLInputElement).value = "";
-        }
-        if (
-            e.key === "Backspace" &&
-            (e.target as HTMLInputElement).value.length === 0
-        ) {
-            setFormData((data: Record<string, any>) => ({
-                ...data,
-                urls: data.urls.slice(0, -1),
-            }));
```

Comment:
The ability to delete the last URL using `backspace` should be kept, not removed